webpack/assets/javascripts/react_app/routes/Audits/AuditsPage/components/AuditsLoadingPage.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import Skeleton from 'react-loading-skeleton';
+import './auditsloading.scss';
+
+const AuditsLoadingPage = () => (
+  <div id="audits-empty-table">
+    <Skeleton count={10} />
```

Comment:
is this too much?  during loading time there's a redundant vertical scrolling bar